sdk/identity/identity/src/msal/browserFlows/browserCommon.ts

```diff
@@ -53,6 +60,48 @@ export function defaultBrowserMsalConfig(
   };
 }
 
+/**
+ * Defines common properties that customizes browser authentication.
+ * The full list of available properties can be seen here: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/3a730d5ad842eaa8fc643015817cd7c20415e921/lib/msal-browser/src/request/PopupRequest.ts
```

Comment:
should this be pinned to 3a730d5ad842eaa8fc643015817cd7c20415e921 or to master?

Reply:
Master might change, so I felt the relevancy of this comment would be accurate until we add more properties 🤔